.changeset/puny-parents-call.md

```
@@ -0,0 +1,14 @@
+---
+"@lynx-js/react": minor
+---
+
+Fix the error when using dynamic `key` with spread attributes on `<list-item>`.
+
+**Error Message:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)